### PR TITLE
Deprecating packr

### PR DESCRIPTION
**Pull request by sio4**
As a part of gobuffalo/buffalo#2152, modernize it.

* updated gobuffalo family module dependencies
* migrated from `packr` to `go:embed`
* fixed command set and updated template
* fixed Github action files: release versioning and test against recent Go versions

### CODE REVIEW

File: genny/auth/templates/actions/auth.go.tmpl [hunk 1]
```diff
@@ -12,8 +12,8 @@ import (
 	{{range .providers -}}
 	"github.com/markbates/goth/providers/{{ downcase . }}"
   {{end -}}
-	"github.com/gobuffalo/pop"
-	"github.com/gobuffalo/pop/nulls"
+	"github.com/gobuffalo/pop/v5"
```
Comment (fasmat) [inline genny/auth/templates/actions/auth.go.tmpl]: Don't forget to update here (and some other places) to `v6` before merging!

**Comment by fasmat** [comment]
Looks good to me

**Comment by sio4** [comment]
@paganotoni can you please merge this?

**Review by fasmat — APPROVED** [review]

File: genny/auth/templates/actions/auth.go.tmpl (hunk 1 above)
Comment (sio4) [reply genny/auth/templates/actions/auth.go.tmpl]: Yeah, it seems like the part is on outdated commit. Thanks!

**Pull request merged**